pkg/controllers/dnspolicy/dnspolicy_controller.go

```diff
@@ -235,3 +238,18 @@ func (r *DNSPolicyReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		).
 		Complete(r)
 }
+
+func (r *DNSPolicyReconciler) ReconcileTTL(policy *v1alpha1.DNSPolicy) {
+	if policy.Spec.TTL == nil {
+		policy.Spec.TTL = &v1alpha1.DNSPolicyTTL{
+			Value: v1alpha1.DNSPoliciesDefaultTTL,
+		}
+		return
+	}
+
+	if policy.Spec.TTL.Value <= 0 {
+		policy.Spec.TTL.Value = v1alpha1.DNSPoliciesDefaultTTL
+	}
+
+	return
+}
```

Comment:
Seems odd to change the spec of the policy in this way

Reply:
good point, I'll update the CRD with a default.